.golangci.yaml

```diff
@@ -70,5 +70,5 @@ linters-settings:
     lines: 120
     statements: 80
 
-  skip-dirs:
```

Comment:
This is not part of the spec: https://golangci-lint.run/usage/configuration/#linters-settings-configuration

Reply:
`skip_dirs` is documented there 🤔

Reply:
Yes, but it's not part of the` linters-settings` object, it's part of the run: https://genai.stackexchange.com/questions/550/how-to-get-two-openai-custom-gpts-talk-with-each-other